tests/unit/components/radio-button-test.js

```diff
@@ -109,6 +109,33 @@ test('it gives the label of a wrapped checkbox a `checked` className', function(
   assert.equal(this.$('label').hasClass('blue-radio'), true, 'has class `blue-radio`');
 });
 
+test('it gives the label of a wrapped checkbox the user supplied `checked` className', function(assert) {
+  assert.expect(4);
+
+  this.set('groupValue', 'initial-group-value');
+  this.set('value', 'component-value');
+
+  this.render(hbs`
+    {{#radio-button
+        groupValue=groupValue
+        value=value
+        checkedClass="active"
```

Comment:
instead of using "active" in these tests, how about something like "my-custom-checked-class" which is clearly user-provided